test/client.spec.js

```diff
@@ -0,0 +1,285 @@
+import core from "@actions/core";
+import webapi from "@slack/web-api";
+import { assert } from "chai";
+import Client from "../src/client.js";
+import Config from "../src/config.js";
+import send from "../src/send.js";
+import { mocks } from "./index.spec.js";
+
+describe("client", () => {
+  beforeEach(() => {
+    mocks.reset();
+  });
+
+  describe("inputs", () => {
+    it("requires a method is provided in inputs", async () => {
+      /**
+       * @type {Config}
+       */
+      const config = {
+        core: core,
+        inputs: {
+          token: "xoxb-example",
+        },
+      };
```

Comment:
These tests skip the standard setups that pass through the `config` constructor to check edge cases **within the module** - this is for both correctness and coverage 👾